Mudit_Gupta/progress.md

```diff
@@ -3,78 +3,80 @@
 **Project: Usability Hero**
 
 Overall Direction:
-- 
-- 
-- 
+- Provide Context specific help
+- Improve Getting Started guide
+- Improve documentation structure
+- Sidebar for sections of instructor help
+- Resolve UI/UX issues
 
 ---
 
 Week 1 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Work on [Student: feedback submission: distribution points question: explain if equal points are not allowed](https://github.com/TEAMMATES/teammates/issues/8817)
+- [ ] Discuss with Cara further work to be done on Getting started guide and help docs
 
 Week 2 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Work on [New instructors: set the time zone of the sample course to match user](https://github.com/TEAMMATES/teammates/issues/8787)
+- [ ] Front end development for sidebar of instructor help
 
 Week 3 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Instructor: Edit session: Rubric question: add front-end validation for empty weights (issue #8631)
+- [ ] Write tests for sidebar
 
 Week 4 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Buffer period to resolve pending work from week 1-3
+- [ ] Fix issues after code review
 
 Week 5 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Develop use cases/user stories + UI workflows for enhancements to Getting started guide
+- [ ] Mockups for enhancements to Getting started guide
 
 Week 6 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Front end development for enhancements to Getting started guide
+- [ ] Writing Selenium tests
```

Comment:
i think the tests gonna take quite long to write and review... the starting guide is really a lot of code i suppose...